### ISSUE TITLE
vdf.parse: one too many closing parenthasis

### ISSUE THREAD

**Issue by PolygonError** [lasa01/io_import_vmf#77]
When I'm importing a map, the process will fail due to a material apparently having one too many closing parenthesis, but there doesn't appear to be any extra or missing parentheses. It doesn't skip the material (which I thing should be added with a warning possibly?), just stops.

Here is the error from the addon
```
Python: Traceback (most recent call last):
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\__init__.py", line 744, in execute
    importer.load(self.filepath, context, self.map_data_path)
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\import_vmf.py", line 147, in load
    self._stage_solid(solid)
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\import_vmf.py", line 482, in _stage_solid
    if self.skip_tools and all(self._vmt_importer.is_nodraw(material, getter) for material, getter in materials):
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\import_vmf.py", line 482, in <genexpr>
    if self.skip_tools and all(self._vmt_importer.is_nodraw(material, getter) for material, getter in materials):
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\import_vmt.py", line 1098, in is_nodraw
    builder = _MaterialBuilder(self._vtf_importer, truncated_name, vmt_data(),
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\__init__.py", line 672, in get_material
    return VMT(vmt_f, self.fs, allow_patch=allow_patch)
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\vmt.py", line 132, in __init__
    vdf_dict: dict = vdf.load(file, escaped=False)
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vdf\__init__.py", line 208, in load
    return parse(fp, **kwargs)
  File "C:\Users\poly\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vdf\__init__.py", line 123, in parse
    (getattr(fp, 'name', '<%s>' % fp.__class__.__name__), lineno, 0, line))
  File "E:\User\Documents\Projects\Source Engine\Projects\Misc\psx_autumn2001\decompiled\psx_autumn2001_d\materials\psx_nature\canalwater001a.vmt", line 15
    }
    ^
SyntaxError: vdf.parse: one too many closing parenthasis

location: <unknown location>:-1
```

And here is the apparently offending VMT it fails on.
`canalwater001a.vmt`
```
"LightmappedGeneric"
{
	"$basetexture" "psx_nature/canalwater001a"
	"$surfaceprop" "mud"
	"%compilepassbullets" "1"
	"Proxies"
	{"TextureScroll"
		{
			"texturescrollvar" 	"$baseTextureTransform"
			"texturescrollrate" 	"0.05"
			"texturescrollangle" 	"90"
			"texturescale" 		"1"
		}
	}
}
```

**Comment by lasa01** [comment]
The vdf parser has some design issues. The issue is the part `{"TextureScroll"`, the parser is unable to read the opening bracket from there. To work around it you can put the `{` on a separate line (by adding a newline between the `{` and `"`).

**Comment by lasa01** [comment]
This is fixed in Plumber 1.0.0 beta versions, you can try the latest version [here](https://github.com/lasa01/io_import_vmf/releases/tag/v1.0.0-beta3).
